app/javascript/application.js

```diff
@@ -0,0 +1,14 @@
+// Entry point for the build script in your package.json
+import "@hotwired/turbo-rails"
+import "./controllers"
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function App() {
+    return (<h1>Hello World!</h1>);
+}
+
+ReactDOM.render(
+    <App/>,
+    document.getElementById('roo'),
+);
```

Comment:
Nice Job setting up the endpoint of the API 😉. there's still an issue with your app😉. currently, your random greetings have still not been rendered. a blank screen is rendered with some errors in the console😉. Apart from this error, don't forget you haven't imported the `greetings` component and used your react-router properly😉.

Kindly see to the issues and do well to fix it as it is a requirement that our react app is properly integrated and works well😉.

<img width="293" alt="image" src="https://github.com/SFarida/hello-rails-react/assets/64643995/62a68124-c6b9-4e03-b661-21beb13cd883">

Reply:
Hi @Ibizugbe  thank you for the review and the hint, it was not easy to resolve but I finally did it, thanks.